jmh-core/src/main/java/org/openjdk/jmh/util/DelegatingMultiset.java

```diff
@@ -105,4 +105,12 @@ public int hashCode() {
         result = 31 * result + (int) (size ^ (size >>> 32));
         return result;
     }
+
+    @Override
+    public void clear() {
+        size = 0;
+        if (map != null) {
```

Comment:
I don't think `map` can be `null` here. You can skip the null check.

Reply:
Fixed.